packages/nvidia_nat_langchain/src/nat/plugins/langchain/llm.py

```
@@ -366,3 +367,288 @@ async def _agenerate(self,
+                    prompt_parts.append(f"User: {message.content}")
+                elif isinstance(message, AIMessage):
+                    prompt_parts.append(f"Assistant: {message.content}")
+                else:
+                    prompt_parts.append(str(message.content))
+            return "\n".join(prompt_parts) + "\nAssistant:"
+
+        def _generate(
+            self,
+            messages: list[BaseMessage],
+            stop: list[str] | None = None,
+            run_manager: CallbackManagerForLLMRun | None = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `run_manager`

(ARG002)